### ISSUE TITLE
Undefined: contrib.NewSIFT when run examples

### ISSUE THREAD

**Issue by mingsterism** [hybridgroup/gocv#863]
## Description

The contrib is always  undefined when i run the test code below. 
```
package main

import (
	"fmt"
	"gocv.io/x/gocv/contrib"
)

func main() {
	si := contrib.NewSIFT()
	fmt.Println(si)

}

```

## Error 
`./testContrib.go:9:8: undefined: contrib.NewSIFT`

## Steps to Reproduce
I've installed according to the documentation 

1.  `make deps`
2. `make download`
3. `make build BUILD_SHARED_LIBS=ON`
4. `make sudo_install`

## Your Environment

```
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```

* Operating System and version:
* OpenCV version used:
* How did you install OpenCV?
* GoCV version used:
* Go version:

```
go run cmd/version/main.go
gocv version: 0.27.0
opencv lib version: 4.5.2
```
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`?
no

**Comment by deadprogram** [comment]
Are you sure you have added to your projects modules by running `go get -u -d gocv.io/x/gocv` when in your project directory?

**Comment by deadprogram** [comment]
I have cleaned up the examples and updated the make task in the `dev` branch.

Marking to close on next release. Thanks for reporting @mingsterism

**Comment by deadprogram** [comment]
Released in v0.28 so now closing. Thanks!
